tests/torch/test_pytorch_patch.py

```diff
@@ -0,0 +1,28 @@
+from nncf.torch.dynamic_graph.patch_pytorch import MagicFunctionsToPatch
+from nncf.torch.graph.operator_metatypes import PT_OPERATOR_METATYPES
+
+
+def test_is_any_metatype_op_names_duplicated():
+    function_name_to_operator_metatype = {}
+    for operator in PT_OPERATOR_METATYPES.registry_dict:
+        for function_name in PT_OPERATOR_METATYPES.get(operator).get_all_aliases():
+            function_name_to_operator_metatype[function_name] = function_name_to_operator_metatype.get(function_name,
+                                                                                                       [function_name])
```

Comment:
Please look carefully over this line, ponder on what it actually does and see that there will never be any duplicates in the underlying dict. Anyway, instead of using this error-prone approach, I recommend using `defaultdict(list)` for the `function_name_to_operator_metatype`.

Reply:
Done